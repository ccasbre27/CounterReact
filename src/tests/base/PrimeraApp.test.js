import React from 'react';
import PrimeraApp from '../../PrimeraApp';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

describe('Pruebas en <PrimeraApp />', () => {
   
    /*
    test('debe mostrar el mensaje "Hola, soy Gokú" ', () => {
       
        const saludo = 'Hola, soy Gokú';
        const { getByText } = render(<PrimeraApp saludo={ saludo }/>)

        expect( getByText( saludo ) ).toBeInTheDocument();

    });
    */

    test('debe de mostrar <PrimeraApp /> correctament', () => {
       
        const saludo = 'Hola, soy Gokú';
        const wrapper = shallow( <PrimeraApp saludo={ saludo } /> );

        expect( wrapper ).toMatchSnapshot()

    });
    
    

});
