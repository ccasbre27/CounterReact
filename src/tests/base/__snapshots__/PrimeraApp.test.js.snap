// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> debe de mostrar <PrimeraApp /> correctament 1`] = `
<Fragment>
  <h1>
     
    Hola, soy Gokú
     
  </h1>
  <p>
    soy un subtítulo
  </p>
</Fragment>
`;
